features/step_definitions/download_endpoint_steps.rb

```diff
@@ -0,0 +1,28 @@
+require 'open-uri'
+require 'tempfile'
+
+Given(/^I try download "([^"]*)" from channel "([^"]*)"$/) do |rpm, channel|
+  url = "#{Capybara.app_host}/rhn/manager/download/#{channel}/getPackage/#{rpm}"
```

Comment:
We set Capybara.app_host from TESTHOST. I prefer to use what is assigned, as it *may* have been sanitized, while TESTHOST is user input.